### ISSUE TITLE
[accessibility] Redundant title text on "improve this map" link

### ISSUE THREAD

**Issue by brandonpeat** [mapbox/mapbox-gl-js#13582]
### mapbox-gl-js version

v3.10.0

### Browser and version

appears in all browsers

----------

Similar to #11036

The "improve this map" hyperlink has visible HTML text, and a `title`, and an `aria-label` that all say the same thing: "Improve this map".

```
<a class="mapbox-improve-map" href="https://apps.mapbox.com/feedback/?owner=XXX" target="_blank" title="Improve this map" aria-label="Improve this map" rel="noopener nofollow">Improve this map</a>
```

My understanding is:
* The `title` text is redundant and not reliably announced by screen readers, and should be removed. It's currently throwing an error in WAVE.
* The `aria-label` is also redundant and should be removed, _unless_ it's used to announce other useful information such as "opens in a new window".

**Comment by brandonpeat** [comment]
So in theory the desired end result would be:

```
<a class="mapbox-improve-map" href="https://apps.mapbox.com/feedback/?owner=XXX" target="_blank" aria-label="Improve this map (opens in new tab)" rel="noopener nofollow">Improve this map</a>
```

**Comment by ibesora** [comment]
Can you provide an example where are you seeing this?
With 3.16.0 the Improve this map link looks like this:

```
<a href="https://apps.mapbox.com/feedback/?..." rel="noopener nofollow" class="mapbox-improve-map">Improve this map</a>
```

Which, as you can see does not contain neither the title nor the aria-label tags.

**Comment by brandonpeat** [comment]
https://drydock-3-dev.asheragencydev.com/demos/location/

I was seeing this on 3.10.0, but just upgraded to 3.16.0 per your comment and still seeing the same thing.

It looks like all three attribution links have the same issue with redundant title and aria, I'm not sure why WAVE is only flagging the Improve link.

**Comment by brandonpeat** [comment]
Hmm, interesting -- I just tested `v3.17.0-beta.1` and that DID fix it on all 3 links. So I guess this will be fixed soon when 3.17.0 is officially released?

**Comment by ibesora** [comment]
Yes, that will happen in a month approximately.
Let's reopen this issue if that's still a problem in the future.
Thanks for reporting it!
